base_name_search_improved/models/ir_model.py

```diff
@@ -175,11 +193,41 @@ def _search_smart_search(self, operator, value):
 class IrModel(models.Model):
     _inherit = "ir.model"
 
-    add_smart_search = fields.Boolean(
-        help="Add Smart Search on search views",
+    add_smart_search = fields.Boolean(help="Add Smart Search on search views",)
```

Comment:
there is something wrong in that change.
it is making breaking Test.
https://github.com/OCA/server-tools/runs/5059800229?check_suite_focus=true#step:4:121